docs/Concepts/Plugins/Plugins.md

```diff
@@ -46,13 +46,18 @@ geth <other parameters> \
 ## Plugin Integrity Verification
 
 Plugin Central Server can be used to download and verify plugin integrity using [PGP](https://en.wikipedia.org/wiki/Pretty_Good_Privacy).
-The architecture enables the same verification process locally via `--plugins.localverify` and `--plugins.publickey` flags or
-remotely with custom plugin central - reference the [`Settings`](../../HowTo/Configure/Plugins.md) section for more information on how to support custom plugin central.
+The architecture enables the same verification process locally via
+[`--plugins.localverify`](../../Reference/CLI-Syntax.md#pluginslocalverify) and
+[`--plugins.publickey`](../../Reference/CLI-Syntax.md#pluginspublickey) flags or
+remotely with custom plugin central - reference the [`Settings`](../../HowTo/Configure/Plugins.md) section for more
+information on how to support custom plugin central.
 
-If the flag `--plugins.skipverify` is provided at runtime the plugin verification process will be disabled.
+If the flag [`--plugins.skipverify`](../../Reference/CLI-Syntax.md#pluginsskipverify) is provided at runtime the plugin
+verification process is disabled.
```

Comment:
Suggest changing this to, "To disable the plugin verification process, use the `--plugins.skipverify` command line option."

This allows the reader to read the first few words of the sentence and decide whether it's relevant to them or not. There may be a technical term for this sort of sentence structure.